src/main/java/com/minecolonies/coremod/util/text/NonSiblingFormattingTextComponent.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.coremod.util.text;
+
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.TextComponentBase;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.util.text.TextComponentBase'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)